frontend/src/js/route-config/route-config.js

```diff
@@ -317,6 +317,17 @@
             title: "Our Team"
         };
 
+        var update_profile = {
+            name: "web.update-profile",
+            parent: "web",
+            url: "/update-profile",
+            templateUrl: baseUrl + "/web/update-profile.html",
+            title: "Update Profile",
+            controller: 'updateProfileCtrl',
+            controllerAs: 'updateprofile',
```

Comment:
As discussed earlier, please make sure you follow the camel casing. Replace `updateprofile` with `updateProfile` and hence similar changes in the view.